files/fr/web/api/css_font_loading_api/index.html

```diff
@@ -0,0 +1,48 @@
+---
+title: CSS Font Loading API
+slug: Web/API/CSS_Font_Loading_API
+tags:
+  - API
+  - CSSFontLoading
+  - CSSOM
+  - Experimental
+  - Fonts
+  - NeedsContent
+  - Reference
+translation_of: Web/API/CSS_Font_Loading_API
+---
+<div>{{DefaultAPISidebar("CSS Font Loading API")}}{{SeeCompatTable}}</div>
+
+<p class="summary">L’API CSS Font Loading API fournit des évènements et interfaces pour le chargement dynamique des ressources associées aux polices.</p>
```

Comment:
Straight apostrophes (') rather than curly ones. I'm aware of the typographic difference between them but the current convention goes against them (cf. https://developer.mozilla.org/fr/docs/MDN/Guidelines/Writing_style_guide#punctuation)

Reply:
ah, that's a big difference from our Guidelines at WordPress. I'll try to refrain from using them, then :)